plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -20,11 +20,15 @@ import (
 	"sync"
 	"time"
 
+	"k8s.io/kubernetes/pkg/util"
+
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 const schedulerSubsystem = "scheduler"
 
+var BindingRateObserveInterval = 200 * time.Millisecond
+
```

Comment:
Also - do you think that exposing it every 200ms is necessary? I wouldn't go below 1s...